Update the ReadME file for the np sample (#78)
ballerina-guides/ai-samples: rate-blog-posts-using-natural-functions/README.md

# Blog Review with Natural Functions

> This project demonstrates how to use natural functions in Ballerina, which allow the function to contain instructions in natural language. Such a function is evaluated at runtime with a call to an LLM. The example uses a natural function to analyze blog content to suggest a category and it based on predefined criteria.

Reviewer: ```suggestion
This project demonstrates how to use natural functions in Ballerina, which allow the function to contain instructions in natural language. Such a function is evaluated at runtime with a call to an LLM. The example uses a natural function to analyze blog content to suggest a category and rate it based on predefined criteria.
```

---

## Prerequisites

1. Install **Ballerina 2201.12.0** or higher.  
   Download and install Ballerina from [here](https://ballerina.io/downloads/).
2. Install the **Ballerina extension** for Visual Studio Code.

---

## Steps to Run

### **Option 1: Use the default model (Without LLM keys)**

> This approach is made available to quickly get up and running with natural functions. Note that this is only meant to be used for quick testing, and that for development and production use-cases, you would have to provide your keys.

Reviewer: ```suggestion
This approach is made available to quickly get up and running with natural functions. Note that this is only meant to be used for trying out, and that for development and production use-cases, you would have to provide your keys.
```

#### 1.Login to WSO2 Copilot

- Log in to your WSO2 Copilot account.

#### 2.Configure the Default Model for Natural Functions

- Press `Ctrl + Shift + P` (or `Cmd + Shift + P` on macOS) to open the command palette.
- Search for and select **"Configure Default Model for Natural Functions"**.
- Your API keys will be automatically generated and added to the `Config.toml` file in your project directory.

#### 3. Run the Program

- Open a terminal in the project directory.
- Run the following command:

  ```bash
  bal run
  ```

---

### **Option 2: Manual Configuration**

> 

Reviewer: ```suggestion

You can use one of the two following options to configure the LLM to use.

```

> #### Option 2A: Configure the Model in `Config.toml`

Reviewer: ```suggestion
#### Option 2A: Configure the Model in the `Config.toml` file
```

> - Add the following configuration to your `Config.toml` file for Azure OpenAI (or your preferred LLM provider):

Reviewer: ```suggestion
Add the model configuration to your `Config.toml` file.

For Azure OpenAI
   
...
   
For OpenAI
```

Remove unnecessary bullet points.

  
  ```toml
  [ballerinax.np.defaultModelConfig]
  serviceUrl = "<SERVICE_URL>"
  deploymentId = "<DEPLOYMENT_ID>"
  apiVersion = "<API_VERSION>"
  connectionConfig.auth.apiKey = "<YOUR_API_KEY>"
  ```

> #### Option 2B: Initialize the Model in Code (Optional)

Reviewer: ```suggestion
#### Option 2B: Initialize the Model in the Code
```

> - If you need more control over the model, you can initialize it in your code and pass it as a parameter:

Reviewer: ```suggestion
If you need more control over the model by function, you can add the `np:Context context` parameter which allows setting the model. You can then initialize the model in your code and pass it as an argument to the context parameter.
```
